package.json

```diff
@@ -207,7 +207,7 @@
     "ocaml": "~4.7.0",
     "reason-sdl2": "^2.10.2020",
     "reason-jsonrpc": "1.5.0",
-    "reason-libvim": "onivim/reason-libvim#3ead103",
+    "reason-libvim": "onivim/reason-libvim#ec703ce",
```

Comment:
Bringing in the `libvim` update here: https://github.com/onivim/reason-libvim/pull/110